src/main.cpp

```diff
@@ -1448,6 +1456,9 @@ bool CScriptCheck::operator()() {
 
 bool CheckInputs(const CTransaction& tx, CValidationState &state, const CCoinsViewCache &inputs, int per_input_adjustment, bool fScriptChecks, unsigned int flags, bool cacheStore, std::vector<CScriptCheck> *pvChecks)
 {
+    // Check for activation of rule changes
+    const bool protocol_cleanup = (flags & SCRIPT_VERIFY_PROTOCOL_CLEANUP);
```

Comment:
For consistency, I suggest you do `(flags & SCR...) != 0`, like in the interpreter code. C++ does have automagic conversion between integer values and bools, so it shouldn't really matter, I guess.

```suggestion
    const bool protocol_cleanup = (flags & SCRIPT_VERIFY_PROTOCOL_CLEANUP) != 0;
```